src/i18n/fr/ui.ts

```diff
@@ -18,10 +18,13 @@ export default UIDictionary({
 	'rightSidebar.a11yTitle': 'Table des matières',
 	'rightSidebar.onThisPage': 'Sur cette page',
 	'rightSidebar.overview': 'Vue générale',
-	'rightSidebar.more': 'Plus',
+	'rightSidebar.community': 'Communauté',
+	'rightSidebar.joinDiscord': 'Rejoignez nous sur Discord',
+	'rightSidebar.readBlog': 'Lisez nos articles',
```

Comment:
```suggestion
	'rightSidebar.readBlog': 'Lis nos articles',
```

Same question about formality here.

Reply:
"Lire nos articles" then also

Reply:
OK, so:
```suggestion
	'rightSidebar.readBlog': 'Lire nos articles',
```